src/diggler/bot.d

```diff
@@ -47,6 +53,7 @@ final class Bot
 
 	package:
 	Admin[] adminList; // Sorted
+	HelpCommand help;
```

Comment:
This is only used in `DefaultCommands`, so it should probably be stored there instead